20230810_foregrounds_cmb/common.toml

```diff
@@ -0,0 +1,9 @@
+lmax_over_nside = 2.5
+healpix = true
+car = false
+output_folder = "output/{tag}"
+output_filename_template= "litebird_{tag}_uKCMB_{telescope}-{band}_nside{nside}.fits"
+unit = "uK_CMB"
+num = 0
+output_reference_frame = "G"
```

Comment:
@giuspugl CMB-S4 is simulated in C coordinates, I assume Litebird uses "G", is it right? however I am wondering if for this work having same coordinates would help.

Reply:
actually maps are so small I can just do both coordinate frames.

Reply:
yeah this seems a very sensible topic,  should this   be discussed and decided at large ?

Reply:
@giuspugl I executed equatorial, it is only 21G total, I'll just do both